.changeset/curly-rabbits-create.md

```
@@ -0,0 +1,5 @@
+---
+"starlight-nextjs-theme": major
+---
+
+Fix width of banner being to large when ToC is not visible
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)